### ISSUE TITLE
How you guys thought about using GitHub's wiki system?

### ISSUE THREAD

**Issue by ipatch** [JABedford/Front-End-Bible#3]
Why don't you guys use GitHub's wiki system to post all your information?

I've been using it for various projects, and it integrates well with the main repo as a git submodule.  As the GitHub wiki is a git repo in upon itself.

You can check how I use it here 👉 [click me](https://github.com/ipatch/dotfiles)

cheers 🍻
Chris

**Comment by jameesy** [comment]
Cheers for the suggestion Chris.

Bare with me, as I haven't used the Wiki feature in itself. It looks like a nice feature.. 
What did you have in mind in this case? A page of a long list of features do you think? In your opinion what would work best?

Many thanks,
James.

**Comment by ipatch** [comment]
@JABedford Well, the wiki feature is nice because you get a customizable _footer_ and _sidebar_  IMHO, I think works better than jam packing everything within a _README.md_.

I'll attach a picture of what I'm talking about.
<img width="1031" alt="screen shot 2018-02-26 at 11 21 19 am" src="https://user-images.githubusercontent.com/613805/36684867-4430ebc2-1ae7-11e8-92c8-d8f17b1ece6d.png">

Custom footer 👇
<img width="746" alt="screen shot 2018-02-26 at 11 22 13 am" src="https://user-images.githubusercontent.com/613805/36684918-69efdec2-1ae7-11e8-8c37-44e2568aa407.png">

And as I suggested in my previous comment make your _wiki_ a a git submodule of your main wiki.

cheers 🍻
Chris

**Comment by jeremybbauer** [comment]
Now that I've read up on GitHub wikis more, I can see it being useful, especially for including screenshots or short gif recordings to go with code snippets so people can see what the code will look like on the web. 

@JABedford Do you mind if I take a crack at it? I'd mostly be copying some of the great stuff you added to the README.md.

**Comment by jameesy** [comment]
Absolutely Jeremy, if you don't mind having a go! I unfortunately have been a bit busy at work the last couple of days, and my progress has halted somewhat! 

Cheers for the suggestion Chris! Its a good one, and all the suggestions and help is massively appreciated! 

Share with anyone who you think might like to get involved. I have given it a fair shout on freecodecamp, and posted on my Instagram. If there is anywhere else you guys are on then give it a crack.

**Comment by jeremybbauer** [comment]
Hey, @JABedford . I wrote up some wiki pages for this. Since I'm not a collaborator on this repo, though, I can't add the wiki pages to it (wrote them in my own forked version of Front-End Bible). If you want, you could add me as a collaborator and I can add the pages myself (totally okay if you aren't comfortable with this 👌), or I can just give you the Markdown files. Let me know what you prefer!

**Comment by jameesy** [comment]
invite sent Jeremy :)

I appreciate this massively buddy!

**Comment by jeremybbauer** [comment]
Awesome! Thanks, @JABedford . 

I've added the pages Home, Tips & Tricks, Learning Resources, and Interviews. On Tips & Tricks, I made sections for CSS and JavaScript. For each Tips & Tricks file, I made a dropdown list with sections for a description, link to the file, and an example. 

I haven't added anything to the Interviews and Learning Resources pages yet. 

I'm not sure if this is the best way to use the wiki, but let me know what you think.
